qiskit/pulse/common/timeslots.py

```diff
@@ -197,6 +192,28 @@ def shifted(self, time: int) -> 'TimeslotCollection':
         slots = [Timeslot(slot.interval.shifted(time), slot.channel) for slot in self._timeslots]
         return TimeslotCollection(slots)
 
+    def begin_time(self, default: int = None) -> int:
```

Comment:
I thought we had settled on `start_time` and `end_time`, or is this just waiting for #2115 to go through?